kbase-extension/static/kbase/js/common/unodep.js

```diff
@@ -108,8 +62,6 @@ define([], () => {
     }
 
     return {
-        formatElapsedTime: formatElapsedTime,
-        formatTime: formatTime,
-        isEqual: isEqual,
+        isEqual,
```

Comment:
All that's left now in this module is `isEqual`, which should be combined with the `lang` module, as I think that's intended to adjust some language features?